pkg/assembler/graphql/schema/hasSBOM.graphql

```diff
@@ -28,9 +29,11 @@ Collector - the GUAC collector that collected the document that generated this a
 Note: Only package object or source object can be defined. Not both.
 
 """
+
+union PkgSrcObject = Package | Source
+
```

Comment:
We should move these below `HasSBOM`. If you look at `model/nodes.go`, the `HasSBOM` go struct lost its comments because we no longer have it attached.